src/Mobs/Enderman.cpp

```diff
@@ -149,6 +149,29 @@ void cEnderman::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		return;
 	}
 
+	if (m_EMState != CHASING)
+	{
+		cMonster * EndermiteFound = GetMonsterOfTypeInSight(mtEndermite, 64);
```

Comment:
This is potentially quite an expensive operation in e.g. a mob farm with lots of endermen about. Not sure we can do anything about it though.

Reply:
Yeah. The only idea that I have, is to cache the result per chunk. But this is kind of expensive and only an edge case.

Reply:
Alright, let' merge and get it done, we can always revisit if there turns out to be a more serious problem.